src/Microsoft.ServiceFabric.Actors/Runtime/ActorStateProviderHelper.cs

```diff
@@ -164,7 +164,7 @@ internal ActorStateProviderHelper(IActorStateProviderInternal owner)
 
                 await Task.Delay(effectiveRetryDelay, userCancellationToken);
 
-                // Reset effective retry delay to orginal value.
+                // Resets effective retry delay to orginal value.
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.